docs/products/postgresql/howto/pg-long-running-queries.rst

```diff
@@ -0,0 +1,87 @@
+Detect and terminate long-running queries
+=========================================
+
+Aiven does not terminate any customer queries even if they run indefinitely, but long-running queries can cause issues by locking resources and therefore preventing database maintenance tasks.
+
+To identify and terminate such long-running queries, you can do it from either:
+
+* `Aiven Console <http://console.aiven.io>`_
+* `PostgreSQL® shell <https://www.postgresql.org/docs/current/app-psql.html>`_ (``psql``)
+
+
+Terminate long running queries from the Aiven console
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In the `Aiven Console <http://console.aiven.io/>`_, you can go to the **Current Queries** tab for your service.
```

Comment:
This will make the warning about having two Aiven Console links go away
```suggestion
In the `Aiven Console <http://console.aiven.io/>`__, you can go to the **Current Queries** tab for your service.
```

Reply:
added changes